minecraft_servers/porgverse/manifest.json

```diff
@@ -12,6 +12,12 @@
   ],
   "location": {
     "country": "The Netherlands",
+    "city": "Amsterdam",
     "country_code": "NL"
-  }
+  },
+  "discord": {
+    "server_id": 811987858402574366,
+    "rename_to_minecraft_name": true
```

Comment:
Please set this to `false`, as it is only a partner feature and the default is false :)

Reply:
Done